api/src/main/java/org/openmrs/module/eptsreports/metadata/HivMetadata.java

```diff
@@ -17,89 +17,81 @@
+		String uuid = Context.getAdministrationService().getGlobalProperty(EptsReportConstants.GLOBAL_PROPERTY_RETURN_VISIT_DATE_FOR_ARV_DRUG_CONCEPT_UUID); // encounterType_id=6
+		return getConcept(uuid);
 	}
 	
-	// PROGRAMS
-	private String ARTProgram = "efe2481f-9e75-4515-8d5a-86bfde2b5ad3";
+	public Concept getARVPlanConcept() {
+		String uuid = Context.getAdministrationService().getGlobalProperty(EptsReportConstants.GLOBAL_PROPERTY_ARV_PLAN_CONCEPT_UUID);
+		return getConcept(uuid);
+	}
 	
-	private String PTV_ETV = "06057245-ca21-43ab-a02f-e861d7e54593";// 8
+	public Concept getDateOfDelivery() {
+		String uuid = Context.getAdministrationService().getGlobalProperty(EptsReportConstants.GLOBAL_PROPERTY_DATE_OF_DELIVERY_CONCEPT_UUID);
+		return getConcept(uuid);
+	}
 	
+	// Encounter types
 	public EncounterType getAdultoSeguimentoEncounterType() {
-		return getEncounterType(S_TARV_ADULTO_SEGUIMENTO);
+		String uuid = Context.getAdministrationService().getGlobalProperty(EptsReportConstants.GLOBAL_PROPERTY_S_TARV_ADULTO_SEGUIMENTO_ENCOUNTER_TYPE_UUID); // encounterType_id=6
+		return getEncounterType(uuid);
 	}
 	
 	public EncounterType getARVPediatriaSeguimentoEncounterType() {
-		return getEncounterType(S_TARV_PEDIATRIA_SEGUIMENTO);
+		String uuid = Context.getAdministrationService().getGlobalProperty(EptsReportConstants.GLOBAL_PROPERTY_S_TARV_PEDIATRIA_SEGUIMENTO_ENCOUNTER_TYPE_UUID); // encounterType_id
+		                                                                                                                                                         // =
+		                                                                                                                                                         // 9
+		return getEncounterType(uuid);
 	}
 	
 	public EncounterType getARVAdultInitialEncounterType() {
-		return getEncounterType(S_TARV_ADULTO_INICIAL_A);
+		String uuid = Context.getAdministrationService().getGlobalProperty(EptsReportConstants.GLOBAL_PROPERTY_S_TARV_ADULTO_INITIAL_A_ENCOUNTER_TYPE_UUID); // encounterType_id
+		                                                                                                                                                     // =
+		                                                                                                                                                     // 9
+		return getEncounterType(uuid);
 	}
 	
-	private String S_TARV_FARMACIA = "e279133c-1d5f-11e0-b929-000c29ad1d07";
-	
 	public EncounterType getARVPharmaciaEncounterType() {
-		return getEncounterType(S_TARV_FARMACIA);
+		String uuid = Context.getAdministrationService().getGlobalProperty(EptsReportConstants.GLOBAL_PROPERTY_S_TARV_FARMACIA_ENCOUNTER_TYPE_UUID);
+		return getEncounterType(uuid);
 	}
 	
 	public EncounterType getMisauLaboratorioEncounterType() {
-		return getEncounterType(MISAU_LABORATORIO);
-	}
-	
-	public Concept getDateOfDelivery() {
```

Comment:
@ynurmahomed why did you remove these concepts?

Reply:
@nthfloor the concepts moved to L48 and L78.